src/components/Common/UI/HamburgerMenu/HamburgerMenu.tsx

```diff
@@ -0,0 +1,37 @@
+import { useTranslation } from 'react-i18next'
+import { type HamburgerMenuProps } from './HamburgerMenuTypes'
+import { useComponentContext } from '@/contexts/ComponentAdapter/useComponentContext'
+import HamburgerIcon from '@/assets/icons/hamburger.svg?react'
+import { useMenu } from '@/components/Common/UI/Menu/useMenu'
+
+export function HamburgerMenu({
```

Comment:
I'm up for being wrong here, but is HamburgerMenu a menu we think about internally where we're providing the icon button and the menu? I wonder if this belongs in the components/Common folder but not the UI folder.

Reply:
Yeah i think moving it to the root of Common makes sense, that also means it would be subject to the linting we put in place to avoid accidentally importing react aria for stuff like this